indicator-nara/032-berekening-html.Rmd

```diff
@@ -0,0 +1,26 @@
+## De droogte-indicator en haar betrouwbaarheid {#h4-2:droogte-indicator}
+
+### Droogte-indicator {#h4-2-1:droogte-indicator}
+De droogte-indicator is op zich vrij eenvoudig te berekenen door voor elk jaar het gemiddelde te berekenen van de $\text {dagen_droog}$-waarden, cfr. \@ref(h3-3:indicator).
+
+Tabel \@ref(fig:droogte-indicator) geeft de berekende indicator-waarden.
+
+
+```{r droogte-indicator, include = TRUE, fig.cap="Droogte-indicator 1985 - 2018"}
+droogte_ind_j <- dagen_droog %>% 
+  group_by(jaar) %>% 
+  summarise(droogte_ind_j = mean(dag_onder_p05)
+            ) %>% 
+  ungroup()
+
```

Comment:
you can avoid the need for two versions. Have a look at `INBOmd::dyn_table()`. Another options is to use something like this

```
if (interactive() || opts_knit$get("rmarkdown.pandoc.to") == "html") {
# code for interactive (html) output
} else {
# code for static (pdf) output
}
```

Reply:
Thanks for this suggestion (and all others !)
I felt the need because of two reasons: formulas with text and formatting tables 
I renamed the formulas so I could use this code to get all the code in one Rmd.